ui/app/configuration-set-record/directives/record-directives.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+(function () {
+  /* globals _ */
```

Comment:
Same as before, if you add to the .jshintrc you won't need to add this to every file where you use _.